orchestrator/test_runner/src/bootstrapping.rs

```diff
@@ -301,7 +303,7 @@ pub fn create_ibc_channel(hermes_base: &mut Command) {
         "create",
         "channel",
         &get_gravity_chain_id(),
```

Comment:
Hermes was updated to v0.15 for ICA support, some of the interfaces changed and new features need to be set up to enable ICA.